### PR TITLE
Feature/event creation page

### PR DESCRIPTION
**Pull request by trevorfitz0**
## Neccesary checkmarks:

    [] All Tests are Passing in all environments

    [✓] The code will run locally

## Type of change

    [✓] New feature
    [] Bug Fix
    [] Refactor

## Description of Change:

    description: The page where the user can create a new event

## Check the correct boxes

    [✓] This broke nothing
    [] This broke some stuff
    [] This broke everything

## Testing Changes

    [✓] No Tests have been changed
    [] Some Tests have been changed
    [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

    [✓] My code has no unused/commented out code
    [] My code has no binding.pry's
    [✓] I have reviewed my code
    [] I have commented my code, particularly in hard-to-understand areas
    [] I have fully tested my code

### CODE REVIEW

File: src/Components/CreateEvent/CreateEvent.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+.line-break {
+    background-color: var(--secondary-color-white);
+    width: 50.25vw;
+    height: .25vh;
+}
+
+.card {
+    height: var(--section-height);
+    width: var(--section-width);
```
Comment (jfogiato) [inline src/Components/CreateEvent/CreateEvent.css:13]: Loving the use of these variables across the app - it'll make everything so much easier

**Review by Gderibeaux — APPROVED** [review]
Looks good!

File: src/Components/Map/Map.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+//    options
+//   );
+//  }, []);
+ return (
+  <div>
+   <input className="long-input" ref={inputRef} />
+  </div>
+ );
+};
+export default AutoComplete;
```
Comment (jfogiato) [inline src/Components/Map/Map.js:23]: Appreciate you pausing this! I wanna make sure we implement this seamlessly so would rather wait and come back when we have bandwidth.

**Review by jfogiato — COMMENTED** [review]
Looks good! Thanks Trevor!

**Pull request merged**